modules/components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -549,6 +549,10 @@ public JsonObject getPopularAssets(String type, String tenantId, int limit,
 				assets.add(parser.parse(gson.toJson(targetId)));
 			}
 			resultSet.close();
+		} catch (NumberFormatException e){
+			String message = errorMsg + e.getMessage();
```

Comment:
better if this is,
String message = errorMsg+ "; Could not parse the tenant ID to an integer"  + e.getMessage();